pbs_light.go

```diff
@@ -91,6 +92,15 @@ type bidResult struct {
 	bid_list pbs.PBSBidSlice
 }
 
+const DEFAULT_PRICE_GRANULARITY = "med"
```

Comment:
For mobile, we don’t want to make the price granularity configurable through the request. This is because we would like the publisher to be able to update their price granularity configuration without having to re-release their app. 

I was hoping to have a UI where the publisher could set the price granularity config as a setting on their account, but that would have to be built out on prebid site.

@josecu suggested a temporary alternative where we could have a price granularity config file in the prebid-server repo where accounts are mapped to configs and if a publisher wants change their price granularity config, they could make a PR to prebid-server to edit that file.

Is there another solution that you're thinking of? I'm not 100% sure what you mean by add the config objects and update the various cache implementations. We're open to any solution where the publisher can update their price granularity configuration on the server in order to avoid having to re-release their app to change their setting.